srcpkgs/meson/template

```diff
@@ -1,11 +1,10 @@
 # Template file for 'meson'
 pkgname=meson
 reverts="0.52.0_1 0.52.0_2"
-version=0.52.1
-revision=2
+version=0.53.1
+revision=1
 archs=noarch
 build_style=python3-module
-pycompile_module="mesonbuild"
 hostmakedepends="python3-devel python3-setuptools"
 depends="ninja python3-setuptools"
 checkdepends="glib-devel gobject-introspection libsanitizer-devel ninja
```

Comment:
`libsanitizer-devel` isn't available on musl

Reply:
What is the best way to manage this?

Reply:
Something like:

```sh
if [ "$XBPS_TARGET_LIBC" = glibc ]; then
	checkdepends+=" libsanitizer-devel"
fi
```

should work.

IIRC, some tests failed badly without libsanitizer.

Reply:
done